<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgitem</class>
 <widget class="QWidget" name="dlgitem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <widget class="QLineEdit" name="edit_name">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>单位名称：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>区域：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cobox_area">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>华东</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>华南</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>华中</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>华北</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>其他</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>外</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>省份：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>单位类型：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cobox_type">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>企业</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>高校</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>研究所</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>代理</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>外</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>政府机构</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>部队</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>第三方</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>地址：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>城市：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="edit_address">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cobox_province">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>山东</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>浙江</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>江苏</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>安徽</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>上海</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>福建</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>湖北</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>湖南</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>河南</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>江西</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>北京</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>天津</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>河北</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>山西</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>内蒙</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>广东</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>广西</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>海南</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>宁夏</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新疆</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>青海</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>陕西</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>甘肃</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>四川</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>云南</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>贵州</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>西藏</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>重庆</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>辽宁</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>吉林</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>黑龙江</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="edit_city">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>网址：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="edit_url">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>采购情况：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>研究方向：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>备注：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="textEdit_research"/>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="textEdit_buy"/>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="textEdit_remark"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_31">
     <property name="text">
      <string>联系方式：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>350</height>
      </size>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>15</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>16</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>17</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>18</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>19</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>20</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>联系人</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>职位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>电话</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>传真</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>邮箱</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>可能过时</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="labelTip_3">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Add">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>增加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Cancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
  <zorder>tableWidget</zorder>
  <zorder>label_31</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
